Apps/CommonData/src/Common.Data.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="FluentValidation" Version="11.5.2" />
+    <PackageReference Include="Macross.Json.Extensions" Version="3.0.0" />
```

Comment:
Where is this package used?

Reply:
Required by Common.Data.DataSource.
@MikeLyttle